sdk/translation/Azure.AI.Translation.Text/tests/TranslationLiveTests.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Translation.Text.Tests
+{
+    public partial class TranslationLiveTests : TextTranslatorLiveTestBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TranslationLiveTests"/> class.
+        /// </summary>
+        /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
+        public TranslationLiveTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [RecordedTest]
+        public async Task TranslateBasic()
+        {
+            string fromLanguage = "es";
+            string targetLanguages = "cs";
+            string inputText = "Hola mundo";
+            TextTranslationClient client = GetClient();
+            var response = await client.TranslateAsync(targetLanguages, inputText, sourceLanguage: fromLanguage).ConfigureAwait(false);
+
+            Assert.AreEqual(200, response.GetRawResponse().Status);
+            Assert.AreEqual(1, response.Value.Count);
+            Assert.AreEqual(1, response.Value.FirstOrDefault().Translations.Count);
+            Assert.AreEqual("cs", response.Value.FirstOrDefault().Translations.FirstOrDefault().To);
+            Assert.NotNull(response.Value.FirstOrDefault().Translations.FirstOrDefault().Text);
+        }
+
+        [RecordedTest]
+        public async Task TranslateWithAutoDetect()
+        {
+            IEnumerable<string> targetLanguages = new[] { "cs" };
+            IEnumerable<string> inputText = new[] { "This is a test." };
+            TextTranslationClient client = GetClient();
+            var response = await client.TranslateAsync(targetLanguages, inputText).ConfigureAwait(false);
+
+            Assert.AreEqual(200, response.GetRawResponse().Status);
+            Assert.AreEqual(1, response.Value.Count);
+            Assert.AreEqual("en", response.Value.FirstOrDefault().DetectedLanguage.Language);
+            Assert.AreEqual(1, response.Value.FirstOrDefault().DetectedLanguage.Score);
```

Comment:
If I understand correctly, these scores are essentially the "confidence" that the predictive model has that the language was detected correctly. If so, assuming that these predictive models evolve and these values could change, would it be safer to simply assert that the score is greater or equal to 0.0 and less or equal to 1.0, instead of asserting a hard-coded value?

Reply:
Switched to `Assert.GreaterOrEqual(0.5, response.Value.FirstOrDefault().DetectedLanguage.Score);`